float64 radius
---
geometry_msgs/Pose grasping_pose